tensorflow/compiler/tf2tensorrt/convert/convert_nodes_test.cc

```diff
@@ -6905,19 +6907,45 @@ TEST_F(OpConverterTest, ConvertResize) {
   {
     // output dimension is a tensor, should fail.
```

Comment:
Let's capitalize the first letter in this comment to make it consistent with other comments.

Reply:
Fixed